TINY-7006: Fire TableModified event when rows are added via Tab key (#6645)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- The color preview box in the color picker dialog was not correctly displaying the saturation and value of the chosen color #TINY-6952
- The color picker dialog will now show an alert if it is submitted with an invalid hex color code #TINY-2814

> ## 5.7.2 - TBD

Reviewer: This shouldn't be here, as this will be part of 5.8.0 if we're merging to `develop`. I also don't believe we have any plans to do a 5.7.2 at this stage (or I haven't heard about it) so we should just remove this for now and include this fix in the 5.8 changelog.

- [ ] Move changes into the 5.8.0/unreleased section of the changelog

### Fixed

> - Fixed a bug where TableModified event were not fired when adding table row via Tab key #TINY-7006

Reviewer 2: ```suggestion
- Fixed a bug where the `TableModified` event was not fired when adding a table row via the Tab key #TINY-7006
```
